sotodlib/coords/__init__.py

```diff
@@ -4,3 +4,4 @@
     Timer, DEG,
 )
 from . import planets
+from .local import * 
```

Comment:
These function are quite specialized and I'd prefer to keep them in an isolated sub-sub-module.  So:
```
 from . import local
```

Reply:
Done